### PR TITLE
General info danny1

### PR DESCRIPTION
**Pull request by xrviv**

### CODE REVIEW

File: _pages/clients/more-speech.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+tags:
+- social
+- client
+- desktop 
+license: 
+---
+
+# A Nostr Browser in Clojure
+
+![more-speech](/images/more-speech.png)
```
Comment (Giszmo) [inline _pages/clients/more-speech.md:17]: That scares people :D

Don't bother adding images to this repo at this point.

File: _pages/clients/more-speech.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: more-speech
+permalink: /more-speech/
+web: 
+github: unclebob/more-speech
+instances:
+tags:
+- social
+- client
+- desktop
```
Comment (Giszmo) [inline _pages/clients/more-speech.md:11]: We could add a different parameter for platform. Adding this dimension is a good idea to later filter by it.

```suggestion
platforms:
- desktop 
```

File: _pages/clients/futr.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+tags:
+- client
+license: MIT
+---
+
+## nostr client desktop app written in haskell
+
+An appImage file is available under its release. However, we weren't able to deploy it successfully.
+
+![futr](/images/futr.png)
```
Comment (Giszmo) [inline _pages/clients/futr.md:16]: Scary screenshot. Nobody will try it with just this. I'd prefer no screenshot over this.

You don't need to try all clients at this point. Mostly just migrate awesome-nostr data over to my repo and one sentence on what's special about this product.

File: _pages/clients/more-speech.md (hunk 1 above)
Comment (xrviv) [reply _pages/clients/more-speech.md:17]: Noted. Deleting images folder - moving all new work to generalInfo

File: _pages/clients/futr.md (hunk 3 above)
Comment (xrviv) [reply _pages/clients/futr.md:16]: Noted. 
- Moving previous work to generalInfo
- Deleting images folder
- will continue work in generalInfo

**Pull request merged**